packages/vuetify/src/mixins/routable/index.ts

```diff
@@ -19,33 +23,71 @@ export default Vue.extend({
       default: undefined
     } as PropValidator<boolean | undefined>,
     exactActiveClass: String,
+    link: Boolean,
     href: [String, Object],
     to: [String, Object],
     nuxt: Boolean,
     replace: Boolean,
-    ripple: [Boolean, Object],
+    ripple: {
+      type: [Boolean, Object],
+      default: null
+    },
```

Comment:
```suggestion
    } as PropValidator<null | boolean | object>,
```

Reply:
Is this necessarily needed? We do not do this anywhere else that we use Ripple